app/lib/package/backend.dart

```diff
@@ -50,6 +50,8 @@ final maxAssetContentLength = 128 * 1024;
 final _defaultMaxVersionsPerPackage = 1000;
 
 final Logger _logger = Logger('pub.cloud_repository');
+final _validGithubUserOrRepoRegExp =
```

Comment:
Do we have a source for this regexp?

Reply:
We could do:
https://github.com/shinnn/github-username-regex

But it might be fine to just check that the `<org>/<repo>` string only contains a single slashes, and no `#` or something like that... and that neither `<org>` or `<repo>` is empty.

We don't really need to be super careful here, since github will only sign JWTs with the correct values.

Reply:
If I understand the regexp correctly, we may be enabling slightly more than github allows, e.g. `-user/...`. I don't think it is super-important to get this exactly right, we only need to filter out obviously bad input.